.github/workflows/main.yml

```diff
@@ -162,18 +162,11 @@ jobs:
           cp $IMAGE-${{ env.HELM_VERSION }}.tgz $IMAGE-latest.tgz
           gsutil cp $IMAGE-*.tgz gs://$ARTIFACT_BUCKET/$IMAGE/
 
-      - uses: actions/create-release@v1
+      - name: Publish Release
         if: github.ref == 'refs/heads/main'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ env.version }}
-          release_name: ${{ env.version }}
-          body: |
-            Automated release
-            ${{ env.version }}
-          draft: false
-          prerelease: false
+        run: gh release create ${{ env.version }} --verify-tag --title ${{ env.version }} --notes ${{ env.version }}
```

Comment:
I think this will fail on `--verify-tag` when it tries to publish the release as there won't be a tag in github for the current version. Not sure if a tag will get automatically generated by the gh release command  with `--verify-tag` removed or whether we need another step which publishes the tag first.

Reply:
You are correct. I've just done a full test and what you predicted occured. I've removed the --verify-tag now and everything seems good. https://github.com/ONSdigital/ras-rm-mock-eq/actions/runs/7526721069/job/20485429203

Reply:
Looks like `gh release create ` is creating the tag as well as the release, which is good. `-verify-tag` looks like it gets used when the tag is published separately before, but can't think of a good reason we need to publish the tag separately.